packages/sanes-chrome-extension/src/utils/test/e2e.ts

```diff
@@ -27,6 +29,20 @@ export async function createPage(browser: Browser): Promise<Page> {
   return page;
 }
 
+export async function getBackgroundPage(browser: Browser): Promise<Page> {
```

Comment:
Remind me when we do the retrospective feedback once the extension is done, to bring the topic for creating another project sharing all test utilities avoiding duplicated code.

Good job by the way.

Reply:
eeh, I want too not to forget about it :)